roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1163,6 +1167,49 @@ def init_local_facts(self, facts=None):
         self.changed = changed
         return new_local_facts
 
+    def init_in_docker_facts(self, facts):
+        facts['is_containerized'] = os.path.isfile('/run/ostree-booted')
+        deployment_type = facts['common']['deployment_type']
+
+        # TODO: Most of these shouldn't actually make it into the facts
+        docker = dict()
+        if deployment_type in ['enterprise','openshift-enterprise']:
+            docker['master_image_name'] = 'openshift3/ose'
+            docker['ovs_image_name'] = 'openshift3/openvswitch'
+            docker['node_image_name'] = 'openshift3/node'
+            docker['etcd_image_name'] = 'rhel7/etcd'
+        elif deployment_type == 'atomic-enterprise':
+            docker['master_image_name'] = 'aep3/aep'
+            docker['ovs_image_name'] = 'aep3/openvswitch'
+            docker['node_image_name'] = 'aep3/node'
+            docker['etcd_image_name'] = 'rhel7/etcd'
+        else:
+            docker['master_image_name'] = 'openshift/origin'
+            docker['ovs_image_name'] = 'openshift/openvswitch'
+            docker['node_image_name'] = 'openshift/node'
+            docker['etcd_image_name'] = 'rhel7/etcd'
```

Comment:
It should be available without a subscription (for the foreseeable future at least). I need to sync up with the team that build that and see if there's a reason we can't use a centos based image for non OSE/AEP installs.

For now I'm going to switch this to using the full registry url of 'registry.access.redhat.com/rhel7/etcd' as I'm not sure non rhel builds of docker will redirect there. This image may not have been available until yesterday's 7.2 release batch.